www/artists/lbeug/ceramics.html

```diff
@@ -20,27 +20,27 @@
                 </p>
                 <div id="gallery">
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/albertmemorial.jpg"><img src="images/th_albertmemorial.jpg" alt="no alt text"></a>
+                        <a href="images/th_albertmemorial.jpg"><img src="images/th_albertmemorial.jpg" alt="no alt text"></a>
```

Comment:
Just a question for future reference in the Artists' Database: why are these images preceded by "th_"? Lorne Beug doesn't really suggest th, compared to an artist with those as their initials. This isn't really necessary information - I'm just curious, and feel it would be useful information for the future.